src/app/components/token-network/token-network.component.ts

```diff
@@ -305,6 +308,15 @@ export class TokenNetworkComponent implements OnInit, OnDestroy {
         this.page.firstPage();
     }
 
+    mintToken(token: UserToken, component: TokenNetworkActionsComponent) {
+        component.requestingTokens = true;
+        const finishRequest = () => (component.requestingTokens = false);
+        this.raidenService
+            .mintToken(token, this.raidenService.raidenAddress, 1000)
+            .pipe(finalize(finishRequest))
+            .subscribe();
```

Comment:
Most of the time we rely on the polling mechanism, that checks every five seconds to refresh the data. But the next polling might be instant or up to 5 seconds later. Wouldn't it be better to call refresh on the `onNext` of the observable so that if the tokens were successfully minted the value would be displayed instantly on the UI?

Reply:
I did not know about this mechanism, but makes sense. Thank you :)